.github/workflows/codeql.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Harden Runner
         uses: step-security/harden-runner@ebacdc22ef6c2cfb85ee5ded8f2e640f4c776dd5
         with:
-          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
+          egress-policy: audit # TODO: change to 'egress-policy: block' after a couple of runs
```

Comment:
@arjundashrath, as discussed, please remove the `# TODO: change to 'egress-policy: block' after a couple of runs` comment.

Reply:
@varunsh-coder good to go. made the change across both the repos